Dockerfile

```diff
@@ -1,10 +1,10 @@
 ## docker image is intended for development environments only.
 ## to use snowshu in production install the package via pip
-FROM python:3.8-slim-buster
+FROM python:3.9-slim-buster
 
-RUN mkdir /app
```

Comment:
Why switch to `/tmp/app` here?

Reply:
the /tmp folder is added to the "file sharing" by default
this is needed to support the run from container